src/submission/schema/schema-manager.ts

```diff
@@ -361,7 +363,11 @@ namespace MigrationManager {
 
     const migrationAfterDonorCheck = await checkDonorDocuments(migration, newTargetSchema);
 
-    const migrationAfterSubmissionsCheck = await revalidateOpenSubmissionsWithNewSchema(
+    migrationAfterDonorCheck.programsWithDonorUpdates?.forEach(program =>
+      messenger.getInstace().sendProgramUpdatedMessage(program),
```

Comment:
this doesn't throw errors right ?

Reply:
Hmm the only one I can think of is if the instance is undefiend which it shouldn't be. All the asyn functions called inside of `sendMessage` are handled with `.catch`. But I guess better safe then sorry. so I could add catch logic here. Also my bad typo should be `getInstance`, will fix.

Reply:
note that when you call for each for something that returns a promise, it will not await on it

Reply:
I don't think we need to wait on the messages being sent though right? The only reason to wait would be if we wanted to know whether the message reached kafka, but I created a seperate ticket to handle errors if messages don't reach kafka.

Reply:
I think so too, should be safe now after the try catch